MyStore/src/JsonParcer.cpp

```diff
@@ -71,11 +70,8 @@ void JsonParcer::ShowData()
 
 	for (auto& it : m_data)
 	{
-
 		std::cout << "Name: " << it.m_product_name << '\n';
-
 		std::cout << "Color: " << it.m_product_color << '\n';
 		std::cout << '\n';

```

Comment:
Since we have overload of << operator for InfoProduct, we don't need this code. Just use that operator for consistency.